ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -909,3 +916,122 @@ def test_restore_trust_pkg_before_restore(self):
             tasks.install_packages(self.master, ['*ipa-server-trust-ad'])
             self.master.run_command(['ipa-restore', backup_path],
                                     stdin_text=dirman_password + '\nyes')
+
+
+class TestBackupRoles(IntegrationTest):
+    """ipa-backup should only run on replicas with all the in-use roles"""
```

Comment:
Docstrings enclosed in """ are multiline-safe, there is no need for splitting over next line.

Reply:
Thanks, I've fixed it.